public/app/core/angular_wrappers.ts

```diff
@@ -23,6 +23,7 @@ import ReactProfileWrapper from 'app/features/profile/ReactProfileWrapper';
 import { LokiAnnotationsQueryEditor } from '../plugins/datasource/loki/components/AnnotationsQueryEditor';
 import { HelpModal } from './components/help/HelpModal';
 import { Footer } from './components/Footer/Footer';
+import { FolderPicker } from 'app/core/components/Select/FolderPicker';
```

Comment:
Hmm, I would suggest `app/core/components/FolderPicker`

Reply:
we have a bunch of different "pickers" under /Select though

Reply:
Ah, this is core, you are right